Overview.src.html

```diff
@@ -934,6 +934,24 @@ <h4 id=terminology-headers>Headers</h4>
 Unless stated otherwise, it is unset.
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-use-token-binding>use-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+<p class="note no-backref"> This flag indicates whether the User Agent will send its Token Binding
```

Comment:
User agent should be lowercase everywhere. We should also have a reference here for the term "Token Binding Id".